src/tests/test_experiment.py

```diff
@@ -1,19 +1,19 @@
 import os.path
-from configparser import ConfigParser
 
 import pytest
+import yaml
 
 from hivpy.experiment import create_experiment, run_experiment
 
 
 @pytest.fixture
 def sample_experiment_params():
     """Sample experiment parameters for testing."""
-    parser = ConfigParser()
-    parser.read(
-        os.path.join(os.path.dirname(__file__), 'fixtures', 'sample.conf'))
-    print(list(parser.items()))
-    return parser
+    filepath = os.path.join(os.path.dirname(__file__), 'fixtures', 'sample.yaml')
+    with open(filepath, 'r') as sample_file:
+        sample_config = yaml.safe_load(sample_file)
+    print(sample_config)
```

Comment:
Oops, this was probably left in from debugging the config parser in the past!

Reply:
Okay I will remove it!